### PR TITLE
vagrant file, bootstrap,GIVIT Django App

### PR DESCRIPTION
**Pull request by nadavshoshan**
- vagrant file to UBUNTU/bionic64
-GIVIT Django app with HelloWorld page

### CODE REVIEW

File: bootstrap.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+echo "install python and pip3"
+apt update
+apt install -y python3.7 python3-pip
+pip3 install --upgrade pip
+pip3 install django==3.1
+
+
```
Comment (lioramilbaum) [inline bootstrap.sh:8]: Remove the extra newline

**Comment by nadavshoshan** [comment]
hey
I deleted Ignored the pyc files and deletes empty lines in bootstrap.sh

**Comment by liorsicd** [comment]
@lioramilbaum can you open the option to add reviewers?

**Comment by nadavshoshan** [comment]
I'll try after the lecture

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,13 @@
+.vagrant/machines/default/virtualbox/action_provision
```
Comment (lioramilbaum) [inline .gitignore:1]: Hard coding the file names is not a good practice nor it is useful. What if a new vagrant version introduce a new file name in the cache directory?

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+.vagrant/machines/default/virtualbox/action_set_name
+.vagrant/machines/default/virtualbox/box_meta
+.vagrant/machines/default/virtualbox/creator_uid
+.vagrant/machines/default/virtualbox/id
+.vagrant/machines/default/virtualbox/index_uuid
+.vagrant/machines/default/virtualbox/private_key
+.vagrant/machines/default/virtualbox/synced_folders
+.vagrant/machines/default/virtualbox/vagrant_cwd
+.vagrant/rgloader/loader.rb
+ubuntu-bionic-18.04-cloudimg-console.log
```
Comment (lioramilbaum) [inline .gitignore:11]: Same idea different file type

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+GIVIT\GIVIT\__pycache__
```
Comment (lioramilbaum) [inline .gitignore:12]: same idea different file/folder type

Comment (nadavshoshan) [reply .gitignore:12]: I just want to make sure that I understand:
1. In order to delete pyc files, I have to delete them from my sync directory and push again.
2. from now on, write the whole directory in the gitignore and not each file?
thanks!
sorry for the poor PR
Nadav

**Comment by lioramilbaum** [comment]
@liorsicd @nadavshoshan Write permissions were added. You can now assign PRs/issues and PR reviewers. Let me know who else (their GH username).

**Comment by nadavshoshan** [comment]
I changed the gitignore. in addition, please give permissions to rotem-benzvi

**Pull request merged**